test/boost/secondary_index_test.cc

```diff
@@ -1533,6 +1533,27 @@ SEASTAR_TEST_CASE(test_computed_columns) {
     });
 }
 
+// Sorted by token value. See testset_tokens for token values.
+static const std::vector<std::vector<bytes_opt>> testset_pks = {
+    { int32_type->decompose(5) },
+    { int32_type->decompose(1) },
+    { int32_type->decompose(0) },
+    { int32_type->decompose(2) },
+    { int32_type->decompose(4) },
+    { int32_type->decompose(6) },
+    { int32_type->decompose(3) },
+};
+
+static const std::vector<int64_t> testset_tokens = {
```

Comment:
Not important here, but I just wanted to comment that what I would have done in a cql-pytest test is to begin the test with a `SELECT token(pk), pk FROM table` and get Scylla to tell me the token so the weird tokens and orderings don't need to be hard-coded in the test. You can see an example of exactly this in test/cql-pytest/test_secondary_index.py, test_partition_order_with_si.